docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe allows you to easily set up testing in different web browsers: a number of [supported browsers](#supported-browsers), or any required browser accessed through [browser provider plugins](#browser-provider-plugins).
+You just need to specify desired browsers in one of two ways:
+
+* through the command line (see [Browser List](../command-line-interface.md#browser-list))
+* programmatically (by using the [runner.browsers](../programming-interface/runner.md#browsers) method)
+
+## Supported Browsers
+
+TestCafe directly supports testing in the following browsers.
+
+Browser           |Version | Short Browser Name
+----------------- | ------ | -------------------
+Chromium          | 22+    | `chromium`
+Google Chrome     | 22+    | `chrome`
+Internet Explorer | 9+     | `ie`
+Microsoft Edge    |        | `edge`
+Mozilla Firefox   | 17+    | `firefox`
+Opera             | 11+    | `opera`
+Safari            | 6+     | `safari`
```

Comment:
So, we've just coined a new term: `directly supported browsers`, as opposed to browsers available through a browser provider. And that's how we are going to introduce it.

First, we should contrast it to 'browser-provider' browsers as early as possible. At that point, the readers have little idea of what we are talking about.

That's why, you are right, we give a link to the corresponding section, but this section should also contain the word `directly` in its name. And the first paragraph is where we can describe it in other words to make it finally clear what we are on about.

Like this:

```
TestCafe allows you to test against different web browsers. The most popular browsers are [supported directly](#directly-supported-browsers), while others require a [browser provider plugin](#browser-provider-plugins).

If you are working with TestCafe through the command line, [list the desired browsers](../command-line-interface.md#browser-list) within the `testcafe` command. In case you are using the API, specify the browsers with the [runner.browsers](../programming-interface/runner.md#browsers) method.

## Directly Supported Browsers

TestCafe supports testing against the following browsers out-of-the-box.
```

Note that I've got rid of meaningless words like `easily` or `just` and wordy constructions: `allows you to set up testing` -> `allows you to test`.

It's also not quite rignt to make an impression like users have a choice how to specify target browsers: through the command line or API. They first choose the interface according to their needs and then seek for information on how to use it. They don't have a choice at that moment.